pkgs/development/tools/misc/autoconf/default.nix

```diff
@@ -6,10 +6,11 @@
 # files.
 
 stdenv.mkDerivation rec {
-  name = "autoconf-2.71";
+  pname = "autoconf";
```

Comment:
I think we should move this one change into a PR that targets staging. I have the hope that all other rebuilds combined are below 500 packages so we can merge it directly into master.

Reply:
so the main goal is to only have "small" changes, that have big impact merged on `staging`? and try to have the rest merged into `master`